cmake/CPM.cmake

```diff
@@ -132,7 +132,6 @@ if(NOT CPM_DONT_CREATE_PACKAGE_LOCK)
 endif()
 
 include(FetchContent)
-include(CMakeParseArguments)
```

Comment:
why is this line removed?

Reply:
Since CMake 3.4 the module is [implemented natively](https://cmake.org/cmake/help/latest/module/CMakeParseArguments.html). As our minimum version is 3.14, we don't need the import.